docs/book/src/topics/consuming-existing-azure-infrastructure.md

```diff
@@ -0,0 +1,6 @@
+# Consuming Existing Azure Infrastructure
+
+Normally, Cluster API will create infrastructure on Azure when standing up a new workload cluster. However, it is possible to have Cluster API re-use existing Azure infrastructure instead of creating its own infrastructure.
```

Comment:
Can we add a note about how this is only for the Azure Cluster infra (not Azure Machines)?